extensions/http/src/main/java/jolie/net/http/HttpMessage.java

```diff
@@ -220,6 +216,21 @@ public void setStatusCode( int code ) {
 		statusCode = code;
 	}
 
+	public String getMethod() {
```

Comment:
@fmontesi this was to force the check on the implemented verbs

Reply:
I have the same concern as @fmontesi. Why should this method return an empty string? Why the check you are mentioning can not be part of the code using this class (rather than introducing such a fragile method in `HttpMessage`)?

Reply:
I have the same concern as @fmontesi. Why should this method return an empty string? Why the check you are mentioning can not be part of the code using this class (rather than introducing such a fragile method in HttpMessage)?